README.md

```diff
@@ -62,6 +62,55 @@ Interactive mode:
 ```
 
 
+## Development (uv, Python 3.13)
+
+This repo uses [uv](https://github.com/astral-sh/uv) and targets Python 3.13.
+
+- Setup environment: `uv sync --extra dev`
+- Run the CLI: `uv run st_package_reviewer --version`
+- Run tests: `uv run pytest`
+- Lint: `uv run flake8 .`
+- Optional watch mode (loop on fail): `uv run pytest -f`
+- Optional parallel runs: `uv run pytest -n auto`
+
+
 [Sublime Text]: https://sublimetext.com/
 [Package Control]: https://packagecontrol.io/
 [wiki]: https://github.com/packagecontrol/st_package_reviewer/wiki
+
+## Development Workflow
+
+- Tests
+  - Quick run: `uv run pytest -q`
+  - With coverage: `uv run pytest --cov st_package_reviewer --cov tests --cov-report term-missing`
+  - Watch mode (loop on fail): `uv run pytest -f`
```

Comment:
This flag does not show up in `pytest --help`. Does it actually exist? I don't see a mention in the docs either.